src/_pytest/assertion/rewrite.py

```diff
@@ -273,13 +277,10 @@ def get_data(self, pathname: Union[str, bytes]) -> bytes:
         with open(pathname, "rb") as f:
             return f.read()
 
-    if sys.version_info >= (3, 9):
+    if sys.version_info >= (3, 10):
```

Comment:
I agree that this should be changed to `(3, 10)` since `FileReader` isn't needed there